### PR TITLE
Pass the SSL configs

### LINKED ISSUE #2750: Kafka Metrics Integration is not using ssl.certificate_authorities, ssl.certificate and ssl.key
After https://github.com/elastic/sdh-beats/issues/1760 it looks like that Kafka metrics integration is not passing in the handlebars the `ssl.certificate_authorities`, `ssl.certificate` and `ssl.key`. 

The fields are declared in https://github.com/elastic/integrations/blob/main/packages/kafka/manifest.yml#L53 but then are not passed to the input from the [hbs files](https://github.com/elastic/integrations/blob/main/packages/kafka/data_stream/broker/agent/stream/stream.yml.hbs)

They need to be added to all the metrics data streams.

### PR DESCRIPTION
**Pull request by lalit-satapathy**
Pass the SSL configs entered in the UI down to agent config for the following metricsets: broker, consumergroup & partition.

Bumping the version to 1.2.2

Closes #2750

## What does this PR do?
Earlier SSL configs were not passed down to the metricset. Fixed it.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

-

## Screenshots
Tested the changes using elastic-package build and run. The new agent policy with ssl entered from the UI.

      - id: kafka/metrics-kafka.partition-052a4ce9-410c-4939-b082-3d5750892edf
        data_stream:
          dataset: kafka.partition
          type: metrics
        metricsets:
          - partition
        period: 10s
        hosts:
          - 'localhost:9092'
        ssl.certificate: ssl1
        ssl.certificate_authorities: c1
        ssl.key: key1

### CODE REVIEW

**Comment by MichaelKatsoulis** [comment]
@lalit-satapathy you need to sign a [Contributor Agreement](https://www.elastic.co/contributor-agreement)

**Comment by lalit-satapathy** [comment]
> @lalit-satapathy you need to sign a [Contributor Agreement](https://www.elastic.co/contributor-agreement)

Yes signed it today. waiting for the closure from the other signatures.

**Review by mtojek — APPROVED** [review]
It's unfortunate, that there aren't any system tests here. I'm wondering if you can add them as follow-ups as part of the onboarding process.

As long as the CI is happy, this PR looks good to me.

**Comment by mtojek** [comment]
@lalit-satapathy Did you use the same email address for commits as for the CLA?

**Comment by lalit-satapathy** [comment]
> @lalit-satapathy Did you use the same email address for commits as for the CLA?

Updated git config email and pushed the changes.

**Comment by mtojek** [comment]
Feel free to merge this PR!

File: packages/kafka/changelog.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- version: "1.2.1"
+  changes:
+    - description: Pass down the SSH configs
```
Comment (hendry-lim) [inline packages/kafka/changelog.yml:3]:
```suggestion
    - description: Pass down the SSL configs
```

**Comment by lalit-satapathy** [comment]
Resolved Merge conflict with #2784

**Comment by lalit-satapathy** [comment]
Closing this PR since Resolving the Merge-Conflicts mixed-up with rebase changes which causes CLA issue. New PR #2792 created.

**Pull request closed without merging**